aspnetcore/fundamentals/static-files.md

```diff
@@ -65,6 +65,8 @@ A request to `http://<app>/StaticFiles/test.png` will serve the *test.png* file.
 
 [!code-csharp[Main](../fundamentals/static-files/sample/StartupAddHeader.cs?name=snippet1)]
 
+The [HeaderDictionaryExtensions.Append](/dotnet/api/microsoft.aspnetcore.http.headerdictionaryextensions.append) method is available from the [Microsoft.AspNetCore.Http](https://www.nuget.org/packages/Microsoft.AspNetCore.Http/) package. Add `using Microsoft.AspNetCore.Http;` to your *csproj* file if the method is unavailable.
```

Comment:
/dotnet/api/microsoft.aspnetcore.http.headerdictionaryextensions.append doesn't look right

> Add `using Microsoft.AspNetCore.Http;` to your *csproj* file if the method is unavailable.

It's in the all package - right?

Reply:
I got *csproj* on the brain! :cry:

Stand-by ... I'll fix it.

As for the API ref, I think that's correct. I think that `Append` comes from the extensions class: https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.headerdictionaryextensions?view=aspnetcore-2.0